bun/src/committeeIndices.ts

```diff
@@ -20,6 +20,7 @@ export function computeProposerIndex(
 	rounds: number,
 ): number {
 	validateShufflingParams(activeIndices, seed, rounds);
+	const binding = getBinding();
```

Comment:
I cannot set binding at import time so has to do it in every api, otherwise got

```
ReferenceError: Cannot access 'binding' before initialization.
```

Reply:
resolved by top-level await